cluster-sharding/src/test/scala/org/apache/pekko/cluster/sharding/ShardRegionSpec.scala

```diff
@@ -183,4 +215,43 @@ class ShardRegionSpec extends PekkoSpec(ShardRegionSpec.config) with WithLogCapt
     }
   }
 
+  "ExtractEntityId" must {
+    "can be safely share to multiple shards" in {
```

Comment:
"be safely able to share multiple shards" reads better to me

When the test runs it prints:
ExtractEntityId must can ... -- this is bad grammar in English